packages/sensors_plus/sensors_plus/ios/Classes/FLTSensorsPlusPlugin.m

```diff
@@ -123,26 +123,45 @@ @implementation FLTAccelerometerStreamHandlerPlus
 - (FlutterError *)onListenWithArguments:(id)arguments
                               eventSink:(FlutterEventSink)eventSink {
   _initMotionManager();
-  [_motionManager
-      startAccelerometerUpdatesToQueue:[[NSOperationQueue alloc] init]
-                           withHandler:^(CMAccelerometerData *accelerometerData,
-                                         NSError *error) {
-                             CMAcceleration acceleration =
-                                 accelerometerData.acceleration;
-                             // Multiply by gravity, and adjust sign values to
-                             // align with Android.
-                             if (_isCleanUp) {
-                               return;
-                             }
-                             sendTriplet(-acceleration.x * GRAVITY,
-                                         -acceleration.y * GRAVITY,
-                                         -acceleration.z * GRAVITY, eventSink);
-                           }];
+  switch (_motionManager.isAccelerometerAvailable) {
```

Comment:
As far as I know there are no modern iOS devices missing one of sensors like it is for some Android devices nowadays, so let's rollback changes related to iOS platform.

For example, here is a data about devices considered pretty old today:
https://www.quora.com/What-are-the-list-of-sensors-in-each-iPhone-iPad-version